@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -837,9 +837,13 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                 }
 
                 // Repeat the process for all the inner rings (holes).
+                const outerRingWinding = ring.winding;
                 const holes: number[] = [];
-                for (; ringIndex < rings.length && rings[ringIndex].isInnerRing; ++ringIndex) {
-                    const current = rings[ringIndex];
+
+                // Iterate over the inner rings. The inner rings have the oppositve winding
+                // of the outer rings.
+                while (ringIndex < rings.length && rings[ringIndex].winding !== outerRingWinding) {
```

Comment:
I'm worried, that this loop stops on first "subring" that has same winding as ultimate parent ring, whereas old loop processed all rings. Is this intended ?

Reply:
Nevermind, outerloop takes care of processing remaining "outer" rings